### ISSUE TITLE
Import covers from folder without #COVER-Tag in txt (feature request)

### ISSUE THREAD

**Issue by m4akus** [UltraStar-Deluxe/Play#408]
Issue type: Feature request

### Actual behaviour

Currently covers are only displayed if the are defined in the song-.txt file with the #COVER-Tag.

### Expected behaviour

In UltraStar Deluxe Song-Covers are not only accepted in the .txt file but also automaticlly displayed if they are in the same folder as the .txt file and the file-name ends with [CO].

For example: 
AnnenMayKantereit x Parcels - Can’t Get You Out Of My Head [CO].jpg

That would work great with the newly implemented zip-download https://usdb.animux.de/ offers.
The ZIP-File contains a folder for each song which itself contains the song-.txt file and the above mentioned cover.

**Comment by achimmihca** [comment]
I agree. I was thinking about automatically using image files in the same folder as cover.

However, the [CO] suffix feels unusual / not needed to me.

I would instead search for a file named "cover" first, then any other image.
Media players (Aimp, Winamp, VLC) also often show these as album art.

**Comment by achimmihca** [comment]
This has been implemented for the Steam release

**Comment by bohning** [comment]
This could be further enhanced by reading the artwork embedded in the audio file, if available.
